autoload/clap/provider/quickfix.vim

```diff
@@ -0,0 +1,34 @@
+" Author: KITAGAWA Yasutaka <kit494way@gmail.com>
+" Description: List the entries of the quickfix list.
+
+let s:save_cpo = &cpoptions
+set cpoptions&vim
+
+let s:quickfix = {}
+
+function! s:quickfix.source() abort
+  " Ignore quickfix list entries with non-existing buffer number.
+  let qflist = filter(getqflist(), 'v:val["bufnr"]')
+
+  return map(qflist, 's:to_grepformat(v:val)')
+endfunction
+
+function! s:to_grepformat(quickfix) abort
+  let path = bufname(a:quickfix['bufnr'])
+  let line_col = a:quickfix['lnum'].' col '.a:quickfix['col']
+  return path.'|'.line_col.'|'.a:quickfix['text']
```

Comment:
`text` is seemingly trimmed for the quickfix entry, I believe we'd better learn that too.

Reply:
I didn't realize that. Thanks.

> I believe we'd better learn that too.

I agree.